pyrdp/mitm/mitm.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, log: SessionLogger, config: MITMConfig):
         self.x224 = X224MITM(self.client.x224, self.server.x224, self.getLog("x224"), self.state, serverConnector, self.startTLS)
         """X224 MITM component"""
 
-        self.mcs = MCSMITM(self.client.mcs, self.server.mcs, self.state, self.recorder, self.buildChannel)
+        self.mcs = MCSMITM(self.client.mcs, self.server.mcs, self.state, self.recorder, self.buildChannel, self.getLog("mcs"))
```

Comment:
Nothing to change but it might cause a conflict with #117 so we just need to watch out for that